src/codegeneration/PureES6Transformer.js

```diff
@@ -65,5 +66,9 @@ export class PureES6Transformer extends MultiTransformer {
     }
     append(AnnotationsTransformer);
     append(TypeTransformer);
+
+    if (options.modules.toLowerCase() === 'inline') {
```

Comment:
I would skip the toLowerCase here. No need for us to support case insensitive values here.